src/components/list-view/list-view.tsx

```diff
@@ -2,51 +2,61 @@ import React, { PureComponent } from 'react';
 
-// TODO: Remove this when coding the backend
-const data: EventCardProps[] = [
-    {
-        date: 'July 13, 2020 5:30 PM',
-        description: 'This is a description of the protest happening at some time about some thing. ya digs! This is a description of the protest happening at some time about some thing. ya digs!',
-        location: 'Los Angeles, CA',
-        imageSource: {uri: 'https://www.pngrepo.com/png/103442/180/protest.png'},
-    },
-    {
-        date: 'July 16, 2020 5:30 PM',
-        description: 'This is a description of the protest happening at some time about some thing. ya digs! This is a description of the protest happening at some time about some thing. ya digs!',
-        location: 'Simi Valley, CA',
-        imageSource: {uri: 'https://www.pngrepo.com/png/103442/180/protest.png'},
-    },
-    {
-        date: 'July 16, 2020 5:30 PM',
-        description: 'This is a description of the protest happening at some time about some thing. ya digs! This is a description of the protest happening at some time about some thing. ya digs!',
-        location: 'Simi Valley, CA',
-        imageSource: {uri: 'https://www.pngrepo.com/png/103442/180/protest.png'},
-    },
-    {
-        date: 'July 16, 2020 5:30 PM',
-        description: 'This is a description of the protest happening at some time about some thing. ya digs! This is a description of the protest happening at some time about some thing. ya digs!',
-        location: 'Simi Valley, CA',
-        imageSource: {uri: 'https://www.pngrepo.com/png/103442/180/protest.png'},
-    },
-    {
-        date: 'July 16, 2020 5:30 PM',
-        description: 'This is a description of the protest happening at some time about some thing. ya digs! This is a description of the protest happening at some time about some thing. ya digs!',
-        location: 'Simi Valley, CA',
-        imageSource: {uri: 'https://www.pngrepo.com/png/103442/180/protest.png'},
-    },
-    {
-        date: 'July 16, 2020 5:30 PM',
-        description: 'This is a description of the protest happening at some time about some thing. ya digs! This is a description of the protest happening at some time about some thing. ya digs!',
-        location: 'Simi Valley, CA',
-        imageSource: {uri: 'https://www.pngrepo.com/png/103442/180/protest.png'},
-    }
-  ];
+const Parse = require('parse/react-native.js');
+
+function initializeServer() {
+    Parse.setAsyncStorage(AsyncStorage);
+    Parse.initialize(ParseConfig.APP_ID, ParseConfig.JAVASCRIPT_KEY);
+    Parse.serverURL = ParseConfig.SERVER_URL;
+}
+
+async function getEvents() {
+    const Event = Parse.Object.extend('Event');
+    const query = new Parse.Query(Event);
+
+    // Query for the earliest 10 events
+    query.limit(10);
+    query.ascending("dateTime");
+    const results = await query.find();
+
+    return results;
+}
+
+function convertToEventCardProp(event) {
+    const prop: EventCardProps = {
```

Comment:
this should be changed to `EventCardState` so the render function works properly. also props should be passed in from the parent class not created / edited in the current class.